code/game/objects/items/control_wand.dm

```diff
@@ -113,40 +270,51 @@
 	department = "omni"
 	region_access = REGION_ALL_STATION
 
-/obj/item/door_remote/captain
+/obj/item/door_remote/command
 	name = "command door remote"
 	department = "command"
+	response_name = span_comradio("CAPTAIN")
```

Comment:
For bonus points grab these titles from the job datums:
`response_name = span_comradio(/datum/job/captain::title)` or whatever the var is called